src/react/components/pages/prebuiltPredict/prebuiltPredictPage.tsx

```diff
@@ -816,4 +846,57 @@ export class PrebuiltPredictPage extends React.Component<IPrebuiltPredictPagePro
             }
         }
     }
+
+    private handleUpdateRequestURI = () => {
+        const {currentPrebuiltType, predictionEndpointUrl} = this.state;
+        console.log("handleUpdateRequestURI");
+        console.log(predictionEndpointUrl);
+        if (predictionEndpointUrl.includes("?")){
+            const additionalUrl = predictionEndpointUrl.split("?")[1];
```

Comment:
This line will fail if predictionEndpointUrl is something like "https://endpoint?", we need to handle this case.